osu.Game/IO/FileStore.cs

```diff
@@ -130,5 +130,12 @@ public override void Cleanup()
                 }
             }
         }
+
+        /// <summary>
+        /// Moved from osu.Framework.IO.FileSafety.
+        /// </summary>
+        /// <param name="path">The path to standardise.</param>
+        /// <returns>The standardised path using '/' for directory separator.</returns>
+        internal static string PathStandardise(string path) => path.Replace('\\', '/');
```

Comment:
A temporary position not unsuitable for the method to live. May be factored out further.

Reply:
Disagree with this placement. How about just making it an extension method and putting in `ExtensionMethods`?

Reply:
Should be sure, but then this PR must be merged together with updating framework.